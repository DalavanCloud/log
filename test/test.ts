describe('log', () => {
  test('succeeds', () => {
    expect(true).toBe(true);
  });
});
