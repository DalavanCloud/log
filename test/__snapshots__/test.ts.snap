// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`log auxillary 1`] = `
Array [
  "[32mâ“¡[39m  batman",
]
`;

exports[`log auxillary 2`] = `
Array [
  "[31mâ“¡[39m  joker",
]
`;

exports[`log debug, trace 1`] = `
Array [
  "[35mâ“¡  debug:[39m batcave",
]
`;

exports[`log debug, trace 2`] = `undefined`;

exports[`log logger 1`] = `
Array [
  "[34mâ“¡[39m  batman",
]
`;

exports[`log preface 1`] = `
Array [
  "[34mâ“¡[39m  â‹Š batcave â‹‰batman",
]
`;

exports[`log unique 1`] = `
Array [
  "[34mâ“¡[39m  batman",
]
`;

exports[`log unique 2`] = `undefined`;
